packages/vscode-extension/src/commands/UpdateObject.ts

```diff
@@ -31,19 +34,36 @@ export default class UpdateObject extends BaseAction {
 
 		const continueMessage = this.translationService.getMessage(ANSWERS.CONTINUE);
 		const cancelMessage = this.translationService.getMessage(ANSWERS.CANCEL);
-		const override = await window.showQuickPick([continueMessage, cancelMessage], {
-			placeHolder: this.translationService.getMessage(UPDATE_OBJECT.OVERRIDE, scriptId),
+		const yes = this.translationService.getMessage(ANSWERS.YES);
+		const no = this.translationService.getMessage(ANSWERS.NO);
+		const processCanceledMessage = this.translationService.getMessage(UPDATE_OBJECT.PROCESS_CANCELED);
+		const suiteCloudCommand = { scriptid: [scriptId], includeinstances: '' };
+		let overwriteMessage = this.translationService.getMessage(UPDATE_OBJECT.OVERWRITE, scriptId);
+
+		if (UpdateObject.isCustomRecord(scriptId)) {
+			const includeInstancesAnswer = await window.showQuickPick([yes, no], {
+				placeHolder: this.translationService.getMessage(UPDATE_OBJECT.QUESTIONS.INCLUDE_INSTANCES),
+				canPickMany: false,
+			});
+			if (includeInstancesAnswer === yes) {
+				suiteCloudCommand.includeinstances = INCLUDE_INSTANCES;
+				overwriteMessage = this.translationService.getMessage(UPDATE_OBJECT.OVERWRITE_INSTANCES, scriptId);
+			}
```

Comment:
I think we are missing the case were the user doesn't respond to the question or just hit ESC , in that case I think we should stop the command flow. Something like:
```
if (!includeInstancesAnswer) {
  return;
}
```